src/EA.Weee.Web/Areas/Scheme/Controllers/OutgoingTransfersController.cs

```diff
@@ -160,6 +160,27 @@ public async Task<ActionResult> SubmittedTransfer(ReviewTransferNoteViewModel mo
             }
         }
 
+        [HttpGet]
+        public async Task<ActionResult> ViewTransferNote(Guid pcsId, Guid evidenceNoteId, int? selectedComplianceYear, bool? returnToView, string redirectTab)
```

Comment:
We dont actually need to do a new action I dont think, which will save unit tests. There is a View action in place,  which may get renamed at some point.  If you have a look at OutgoingTransfers.cshtml there are custom routes defined that allow the URL to match the content catalog.  This property would just need to be updated... EvidenceNoteRowViewModel.SchemeViewRouteName. This would just allow us to reuse existing actions and unit tests **Actually have you have done below :)**

`context.MapLowercaseDashedRoute(
                name: SchemeTransferEvidenceRedirect.ViewApprovedTransferEvidenceRouteName,
                url: "Scheme/{pcsId}/transfer-evidence/outgoing-transfers/view-approved-transfer/{evidenceNoteId}",
                defaults: new { action = "TransferredEvidence", controller = "TransferEvidence", area = "Scheme" },
                namespaces: new[] { typeof(ManageEvidenceNotesController).Namespace });`